api/rest/api.yaml

```diff
@@ -954,6 +963,14 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/ResourceDto'
+        kernelArguments:
+          type: array
+          items:
+            $ref: '#/components/schemas/KernelArgumentDto'
+        globalWorkShape:
+          type: string
```

Comment:
Czym jest ten shape? Dlaczego string? Jeśli musi być string to trzeba opisać czym on jest

Reply:
przerobione na listę integerów